executor/runtime/docker/docker.go

```diff
@@ -1158,6 +1161,31 @@ func (r *DockerRuntime) createTitusEnvironmentFile(c runtimeTypes.Container) err
 	return writeTitusEnvironmentFile(c.Env(), f)
 }
 
+func getUnameR() (string, error) {
+	cmd := exec.Command("/bin/uname", "-r")
+	cmdOutput := &bytes.Buffer{}
+	cmd.Stdout = cmdOutput
+	err := cmd.Run()
+	if err != nil {
+		return "", nil
+	}
+	output := cmdOutput.Bytes()
+	return strings.TrimSpace(string(output)), nil
+}
+
+func getKernelBindMounts() []string {
+	mounts := []string{
+		"/boot:/boot:ro",
+		"/lib/modules:/lib/modules:ro",
+	}
+	unameR, err := getUnameR()
+	if err != nil {
+		kernelHeaders := fmt.Sprintf("/usr/src/linux-headers-%s:/usr/src/linux-headers-%s:ro", unameR, unameR)
+		mounts = append(mounts, kernelHeaders)
```

Comment:
Will this work with bind mounts, or since the uid is unmapped, will it show up as uid 0? (nobody / unbound?)

Reply:
Shows up as nobody/nogroup, works fine.